support-frontend/assets/components/spreadTheWord/spreadTheWord.jsx

```diff
@@ -14,19 +13,16 @@ export default function SpreadTheWord() {
 
   return (
     <div className="component-spread-the-word">
-      <PageSection
-        modifierClass="spread-the-word"
-        heading="Spread the word"
-      >
-        <p className="component-spread-the-word__description">
-          We report for everyone. Let your friends and followers know that you
-          support independent journalism.
-        </p>
-        <div className="component-spread-the-word__share">
-          <SocialShare name="facebook" />
-          <SocialShare name="twitter" />
-        </div>
-      </PageSection>
+      <h3 className="component-spread-the-word__title">Share your support for independent journalism</h3>
+      <p className="component-spread-the-word__description">
+          Let your friends, family and followers know that you made a valuable contribution to The Guardian today
+      </p>
+      <div className="component-spread-the-word__share">
+        <SocialShare name="facebook" />
+        <SocialShare name="twitter" />
+        <SocialShare name="linkedin" />
+        <SocialShare name="email" />
+      </div>
     </div>
   );
 
```

Comment:
Not for this PR but this whole component might be a change to use the [asyncronous](https://github.com/guardian/support-frontend/blob/master/support-frontend/assets/pages/paper-subscription-checkout/components-ty/thankYou.jsx#L122) loader! it will split it out of the main js which will make the initial load time a bit smaller

Reply:
👍 Good idea, I'll add to TBD above